src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -228,7 +237,7 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
     private _getOptionsScript(
         initializationData: PaypalCommerceInitializationData,
         currencyCode: Cart['currency']['code'],
-        apmMehodId?: string
+        initializationMehodId?: string
```

Comment:
Did you spell it correctly? Should it be `initializationMethodId` instead? 🤔

Reply:
Damn, my fault